core/encoding/json/unmarshal.odin

```diff
@@ -215,6 +215,12 @@ unmarshal_value :: proc(p: ^Parser, v: any) -> (err: Unmarshal_Error) {
 		}
 	}
 	
+    switch dst in &v {
+    // Handle json.Value as an unknown type
+    case Value:
+        dst = parse_value(p) or_return
+        return
+    }
```

Comment:
Please use tabs for indentation.

Reply:
Sorry, totally forgot about that. It should be fixed now.